Common/CompatibilityOptions.cs

```diff
@@ -0,0 +1,97 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Files.Common
+{
+    public class CompatibilityOptions
+    {
+        public OSCompatibility OSCompatibility { get; set; }
+        public ReducedColorMode ReducedColorMode { get; set; }
+        public bool ExecuteAt640X480 { get; set; } // 640X480
+        public bool DisableMaximized { get; set; } // DISABLEDXMAXIMIZEDWINDOWEDMODE
+        public bool RunAsAdministrator { get; set; } // RUNASADMIN
+        public bool RegisterForRestart { get; set; } // REGISTERAPPRESTART
+        public HighDpiOption HighDpiOption { get; set; }
+        public HighDpiOverride HighDpiOverride { get; set; }
+
+        public override string ToString()
+        {
+            var value = $"~ {OSCompatibility.GetDescription()} {ReducedColorMode.GetDescription()} {HighDpiOption.GetDescription()} {HighDpiOverride.GetDescription()} " +
+                $"{(ExecuteAt640X480 ? "640X480" : "")} {(DisableMaximized ? "DISABLEDXMAXIMIZEDWINDOWEDMODE" : "")}" +
+                $"{(RunAsAdministrator ? "RUNASADMIN" : "")} {(RegisterForRestart ? "REGISTERAPPRESTART" : "")}";
```

Comment:
Could you move it to `Constants`? It's being used more than once.

Reply:
In the global "Files.Constant" class or in a Constants class private to CompatibilityOptions? Don't think this values are gonna be used anywhere else.

Reply:
What am I saying, CompatibilityOptions does not belong to the Files project lol